CustomerModel　index更新

ManamiNomura opened the pull request.

ManamiNomura committed:
CustomerModel　index更新

YutaMakidera commented:
確認しました
@EguchiAkira0908 
問題なければマージお願いします

EguchiAkira0908 commented on Model/CustomerModel.php:
```diff
@@ -264,11 +264,26 @@ public function show()
+    } catch (PDOException $Exception) {
+      exit("接続エラー：" . $Exception->getMessage());
+    }
+    return $customer;
```
returnで返す変数は$customers複数形で統一したいかな。

EguchiAkira0908 reviewed (commented):
Modelの283だけ直してくれればおけです。

ManamiNomura committed:
Update CustomerModel.php

283行目　　return $customer;　→　return $customers;　に修正。

returnの変数名、$customers　で統一するけど、
動作確認もあるから、他のcheck  input  iogin は、いったん未修正で出してます。
